app/assets/javascripts/two-pane.js

```diff
@@ -144,6 +144,11 @@ function right_pane_content(response){
   var contentId;
   if (handle_redirect(response)) return; //session expired redirect to login
 
+  // move flash messages from response to main window
+  $('.flash', response).each(function(index, item) {
```

Comment:
I guess I'm missing the value in your change, why is it required? (note that we simply changed the notification div to be inside the content div.

if you think its still required, please update the PR to work against the develop branch. thanks!